src/msg/msg_types.h

```diff
@@ -15,6 +15,10 @@
 #ifndef CEPH_MSG_TYPES_H
 #define CEPH_MSG_TYPES_H
 
+#include <string>
+#include <sstream>
+#include <iostream>
```

Comment:
why `string` and `iostream` are needed for `ostringstream`?

Reply:
@tchaikov 
As per suggestion on stackoverflow..., which seemed reasonable
But you are correct here it looks like it only needs `sstream`
Will rebase.